### PR TITLE
Rebalance nukie mechs

### PR DESCRIPTION
**Pull request by Simplehorror**
# Document the changes in your pull request
Cuts nukie mech prices in half. They're just noob bait against any half-competent crew and get shredded instantly by some assistant with an axe circling it.
# Wiki Documentation
Nukie mech prices are now 25% off
# Changelog
:cl:  
tweak: Nukie mech prices are now 25% off
/:cl:

### CODE REVIEW

**Comment by Simplehorror** [comment]
stop reacting to my PRs

**Comment by Marmio64** [comment]
half is pretty generous, im not sure it needs to be cut by that much

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -1317,21 +1317,21 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
...
+	cost = 40 //Yogs change
 	include_modes = list(/datum/game_mode/nuclear/clown_ops)
 
 /datum/uplink_item/support/mauler
 	name = "Mauler Exosuit"
 	desc = "A massive and incredibly deadly military-grade exosuit. Features long-range targeting, thrust vectoring \
 			and deployable smoke. Comes equipped with an LMG, scattershot carbine, missile rack, an antiprojectile armor booster and a Tesla energy array."
 	item = /obj/mecha/combat/marauder/mauler/loaded
-	cost = 140
+	cost = 70 //Yogscast change
```
Comment (Notamaniac) [inline code/modules/uplink/uplink_items.dm]: yogscast?

**Comment by KillerOrcaCora** [comment]
Nuke mechs are indeed lacking, but I would like it if a more thought-out approach was met like actually working on the mechs themselves aside from just slapping a lower price tag and calling it a day.

**Comment by Simplehorror** [comment]
> Nuke mechs are indeed lacking, but I would like it if a more thought-out approach was met like actually working on the mechs themselves aside from just slapping a lower price tag and calling it a day.

i'm all ears

**Comment by KillerOrcaCora** [comment]
> > Nuke mechs are indeed lacking, but I would like it if a more thought-out approach was met like actually working on the mechs themselves aside from just slapping a lower price tag and calling it a day.
> 
> i'm all ears

Your gonna have to conjure up your own creativity, run a test server and play with mechs or search nukie replays and see why they usually suck and then go off from there

**Comment by githubuser4141** [comment]
I think it's largely player incompetence, and that alt strafe didn't work until a recent PR and also mech armor directions were broken, too, with those two issues solved nukie mechs should be extremely robust in the hands of competent players

**Comment by JamieD1** [comment]
Ill consider 25% off

**Comment by Simplehorror** [comment]
done

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**